pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -84,7 +84,7 @@ func setSuid(path string) error {
 	}
 
 	/* Can't do this before the chown as the chown will reset the suid bit */
-	stdOut, stdErr, err = crcos.RunWithPrivilege(fmt.Sprintf("set suid for %s", path), "chmod", "u+s", path)
+	stdOut, stdErr, err = crcos.RunWithPrivilege(fmt.Sprintf("set suid for %s", path), "chmod", "u+s,g+x", path)
```

Comment:
Hmm this assumes that the file is u+x, this assumes the user running crc is part of the `wheel` group. Not sure we want to address this right now, but `setSuid` could probably still fail in some corner cases.